website/source/docs/auth/approle.html.md

```diff
@@ -131,34 +131,91 @@ token_policies  [default]
 
 ### Via the API
 
-#### Enable AppRole authentication
+#### Enable the AppRole authentication.
 
-```shell
-$ curl -XPOST -H "X-Vault-Token:xxx" "http://127.0.0.1:8200/v1/sys/auth/approle" -d '{"type":"approle"}'
+```javascript
+$ curl -X POST -H "X-Vault-Token:$VAULT_TOKEN" -d '{"type":"approle"}' http://127.0.0.1:8200/v1/sys/auth/approle
 ```
 
-#### Create a role
+#### Create an AppRole with desired set of policies.
 
-```shell
-$ curl -XPOST -H "X-Vault-Token:xxx" "http://127.0.0.1:8200/v1/auth/approle/role/testrole" -d '{"secret_id_ttl":"10m", "token_ttl":"20m", "token_max_ttl":"30m", "secret_id_num_uses":40}'
+```javascript
+$ curl -X POST -H "X-Vault-Token:$VAULT_TOKEN" -d '{"policies":"dev-policy,test-policy"}' http://localhost:8200/v1/auth/approle/role/testrole
 ```
 
-#### Fetch the RoleID of the AppRole
+#### Fetch the identifier of the role.
 
-```shell
-$ curl -XGET -H "X-Vault-Token:xxx" "http://127.0.0.1:8200/v1/auth/approle/role/testrole/role-id"
+```javascript
+$ curl -X GET -H "X-Vault-Token:$VAULT_TOKEN" http://localhost:8200/v1/auth/approle/role/testrole/role-id | jq .
 ```
 
-#### Get a SecretID issued against the AppRole
+```javascript
+{
+  "auth": null,
+  "warnings": null,
+  "wrap_info": null,
+  "data": {
+    "role_id": "988a9dfd-ea69-4a53-6cb6-9d6b86474bba"
+  },
+  "lease_duration": 0,
+  "renewable": false,
+  "lease_id": "",
+  "request_id": "ef5c9b3f-e15e-0527-5457-79b4ecfe7b60"
+}
+```
 
-```shell
-$ curl -XPOST -H "X-Vault-Token:xxx" "http://127.0.0.1:8200/v1/auth/approle/role/testrole/secret-id"
+#### Create a new secret identifier under the role.
+
+```javascript
+$ curl -X POST -H "X-Vault-Token:$VAULT_TOKEN" http://localhost:8200/v1/auth/approle/role/testrole/secret-id | jq .
```

Comment:
`localhost` vs `127.0.0.1` is used intermittently, but also on Debian localhost actually maps to 127.0.1.1. Probably best to just keep it as 127.0.0.1.